### PR TITLE
[DOCS]  Document Batches and Partitioners Functionality for GX Cloud

### PR DESCRIPTION
**Pull request by kwcanuck**
[DSB-475](https://greatexpectations.atlassian.net/browse/DSB-475) requests adding content to the existing GX Cloud documentation that documents how to add partitioners to Data Assets. This PR implements this request. 

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Appropriate tests and docs have been updated

[DSB-475]: https://greatexpectations.atlassian.net/browse/DSB-475?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jcampbell — CHANGES_REQUESTED** [review]
I'd like to request a name change away from "splitters".

Before we lock that in, I need to confirm with the Name Tsar (Hernan)

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md [hunk 1]
```diff
@@ -47,27 +47,33 @@ Create a Data Asset to define the data you want GX Cloud to access. Currently, t
...
 
-    - **Authenticator**: Enter the Snowflake database authenticator that you want to use to verify your Snowflake connection. 
+6. Optional. Make the following changes:
 
-5. Optional. Clear **Create temp table** if you don't want to create a temporary database table. Temporary database tables store data temporarily and can improve performance by making queries run faster.
+    - Clear **Create temp table** if you don't want to create a temporary database table. Temporary database tables store data temporarily and can improve performance by making queries run faster.
 
-6. Optional. Clear **Test connection** if you don't want to test the Data Asset connection. Testing the connection to the Data Asset is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
+    - Select **Test connection** if you don't want to test the Data Asset connection. Testing the connection to the Data Asset is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]:
```suggestion
    - Select **Test connection** if you want to test the Data Asset connection. Testing the connection to the Data Asset is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
```
Shouldn't they be selecting **Test connection** if they do want to test the Data Asset connection, or leave it unselected it if they do not?

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md [hunk 2]
```diff
@@ -47,27 +47,33 @@ Create a Data Asset to define the data you want GX Cloud to access. Currently, t
...
 
 7. Click **Continue**.
 
 8. Select **Table Asset** or **Query Asset** and complete the following fields:
 
-    - **Asset name**: Enter a name for the Data Asset. Data Asset names must be unique. If you use the same name for multiple Data Assets, each Data Asset must be associated with a unique Data Source.
-
     - **Table name**: When **Table Asset** is selected, enter a name for the table you're creating in the Data Asset.
+    
+    - **Data Asset name**: Enter a name for the Data Asset. Data Asset names must be unique. If you use the same name for multiple Data Assets, each Data Asset must be associated with a unique Data Source.
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]:
```suggestion
    - **Data Asset name**: Enter a name for the Data Asset. If you use the same name for multiple Data Assets, each Data Asset must be associated with a unique Data Source.
```
The sentence about Data Asset names needing to be unique is contradicted by the sentence immediately following it.  I think we can cut it out, as the second sentence conveys the point more accurately.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md [hunk 3]
```diff
@@ -47,27 +47,33 @@ Create a Data Asset to define the data you want GX Cloud to access. Currently, t
...
 
     - **Query**: When **Query Asset** is selected, enter the query that you want to run on the table. 
 
-9. Optional. Select **Add another Data Asset** to add additional tables or queries and repeat step 7.
+9. Select the **Complete Asset** tab to create Expectations and run validations on the entire Data Asset, or select the **Batches** tab to create Expectations and run validations on subsets of Data Asset records and then complete the following fields:
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: I think this should be reworded.  They aren't creating Expectations or running validations from either of these tabs, they're determining how the records in the Data Asset will be organized.
```suggestion
9. Select the **Complete Asset** tab to provide all Data Asset records to your Expectations and validations, or select the **Batches** tab to use subsets of Data Asset records for your Expectations and validations. If you selected the **Batches** tab, complete the following fields:
```

File: docs/docusaurus/docs/cloud/quickstarts/snowflake_quickstart.md [hunk 4]
```diff
@@ -86,23 +88,31 @@ Create a Data Asset to define the data you want GX Cloud to access within Snowfl
...
     - **Role**: Enter your Snowflake role.
 
-    - **Authenticator**: Enter the Snowflake database authenticator that you want to use to verify your Snowflake connection. 
+6. Optional. Make the following changes:
 
-4. Optional. Clear **Create temp table** if you don't want to create a temporary database table.
+    - Clear **Create temp table** if you don't want to create a temporary database table. Temporary database tables store data temporarily and can improve performance by making queries run faster.
 
-5. Optional. Clear **Test connection** if you don't want to test the Data Asset connection.
+    - Select **Test connection** if you don't want to test the Data Asset connection. Testing the connection to the Data Asset is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/cloud/quickstarts/snowflake_quickstart.md]:
```suggestion
    - Select **Test connection** if you want to test the Data Asset connection. Testing the connection to the Data Asset is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
```

File: docs/docusaurus/docs/cloud/quickstarts/snowflake_quickstart.md [hunk 5]
```diff
@@ -86,23 +88,31 @@ Create a Data Asset to define the data you want GX Cloud to access within Snowfl
...
+8. Select **Table Asset** or **Query Asset** and complete the following fields:
 
     - **Table name**: When **Table Asset** is selected, enter a name for the table you're creating in the Data Asset.
+    
+    - **Data Asset name**: Enter a name for the Data Asset. Data Asset names must be unique. If you use the same name for multiple Data Assets, each Data Asset must be associated with a unique Data Source.
 
     - **Query**: When **Query Asset** is selected, enter the query that you want to run on the table. 
 
-7. Optional. Select **Add table/query** to add additional tables or queries and repeat step 6.
+9. Select the **Complete Asset** tab to create Expectations and run validations on the entire Data Asset, or select the **Batches** tab to create Expectations and run validations on subsets of Data Asset records and then complete the following fields:
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/cloud/quickstarts/snowflake_quickstart.md]: If you reworded this in the manage_data_assets guide, you should reword it here as well.

**Review by Rachel-Reverie — APPROVED** [review]
Looks good other than a few minor suggestions.  The only ones that I think require a change are the two places where "don't" appears to have been incorrectly left in a statement; take or leave the others.  Marking this as approved so that you aren't waiting on me after you've made updates.

File: docs/docusaurus/docs/cloud/quickstarts/snowflake_quickstart.md (hunk 5 above)
Comment (kwcanuck) [inline docs/docusaurus/docs/cloud/quickstarts/snowflake_quickstart.md]:
```suggestion
9. Select the **Complete Asset** tab to provide all Data Asset records to your Expectations and validations, or select the **Batches** tab to use subsets of Data Asset records for your Expectations and validations. If you selected the **Batches** tab, complete the following fields:
```

Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/quickstarts/snowflake_quickstart.md]: @Rachel-Reverie I added the edit as recommended. Thanks!

**Comment by kwcanuck** [comment]
@jcampbell I removed instances of _splitter_ and _splitters_ and replaced them with _partition_ or _partitions_ where applicable. Please review the changes, provide feedback, or unblock the PR when time permits.

**Review by jcampbell — APPROVED** [review]
Thank you, @kwcanuck . I think we may want to make some additional in-app copy changes ("Split Data by") ... do we have a good way to ensure that this doc will be kept in sync?

**Comment by kwcanuck** [comment]
@jcampbell I perform a final test on procedures before they are released to ensure they are accurate and reflect the existing UI. I also monitor Slack channels for announcements of changes and updates.

**Pull request merged**